### ISSUE TITLE
Custom tutorials

### ISSUE THREAD

**Issue by borisbrdaric** [Wirecloud/wirecloud#51]
It should be possible to define custom user tutorials which are accessed from User menu (upper right corner) -> Tutorials or from the Welcome screen (empty workspace).

This functionality would be used together with 'User template' issue (https://github.com/Wirecloud/wirecloud/issues/50) to provide instructions for user first login.

**Comment by aarranz** [comment]
We were thinking also on allowing to pack tutorials into mashups, and allow users to associate tutorials to dashboards.

**Comment by borisbrdaric** [comment]
Yes, that would be nice. Do you have an estimation when will these features be available?
